pkg/apis/nmstate/v1alpha1/nodenetworkconfigurationpolicy_types.go

```diff
@@ -0,0 +1,64 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// NodeNetworkConfigurationPolicySpec defines the desired state of NodeNetworkConfigurationPolicy
+// +k8s:openapi-gen=true
+type NodeNetworkConfigurationPolicySpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
+	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+
+	// In case of multiple policies applying for the same node this
+	// priority define the order from low to high
```

Comment:
from high to low, right? or the highest to the lowest. we take priority 100 before 1.

Reply:
From the design document 

> The implementation may process NodeNetworkConfigurationPolicy entries based on priority (from low to high), such that each entry may override a previous value in the desired state. It is left up to the implementation to define such behaviour, especially in cases where the priority on an entry is equal with other “colliding” entries.

So it's says we apply policies from low to high priority, so lastest applied will override (or this is what I understand), so 100 will override stuff at 1.

Reply:
Aha! Sorry, that makes sense.